src/main/groovy/com/google/appengine/AppEnginePlugin.groovy

```diff
@@ -378,6 +379,15 @@ class AppEnginePlugin implements Plugin<Project> {
         appengineRollbackTask.group = APPENGINE_GROUP
     }
 
+    private void configureSetDefaultVersionTask(Project project, File explodedAppDirectory) {
+        project.tasks.withType(SetDefaultVersionTask).whenTaskAdded { SetDefaultVersionTask appengineSetDefaultVersionTask ->
+            appengineSetDefaultVersionTask.conventionMapping.map(EXPLODED_WAR_DIR_CONVENTION_PARAM) { explodedAppDirectory }
```

Comment:
@salguarnieri old message seems to have disappeared, but it's some kind of lazy evaluation that gradle uses to ensure reading only after the "configuration" phase is complete